tests/blueprints/test_main_config_bp.py

```diff
@@ -0,0 +1,50 @@
+import os
+from typing import Generator
+
+import pytest
+from flask import Flask, Response
+from flask.testing import FlaskClient
+
+
+@pytest.fixture(scope="session", autouse=True)
+def setup() -> Generator[None, None, None]:
+    original_config_path = os.environ.get("CONFIG_FILE_PATH")
+
+    current_file_path = os.path.abspath(__file__)
+    project_root_dir = os.path.dirname(os.path.dirname(current_file_path))
+    test_config_path = os.path.join(
+        project_root_dir,
+        "example_config_combined.yaml",
+    )
+
+    os.environ["CONFIG_FILE_PATH"] = test_config_path
+    yield
+    if original_config_path:
+        os.environ["CONFIG_FILE_PATH"] = original_config_path
+    else:
+        os.environ.pop("CONFIG_FILE_PATH", None)
```

Comment:
in https://github.com/getsentry/script-runner/pull/101 i ended up removing this env var as it makes things easier to test, and makes the lib more flexible. now the config file path is passed to the `create_flask_app` function with the other app config